src/psyclone/tests/domain/gocean/transformations/globalstoargs_test.py

```diff
@@ -45,7 +45,7 @@
 API = "gocean1.0"
 
 BASEPATH = os.path.join(os.path.dirname(os.path.abspath(__file__)),
-                        "..", "..", "test_files")
+                        "..", "..", "..", "test_files")
```

Comment:
Slightly picky but we shouldn't do ".." to go up a directory. Please use functionality from within `os.path` to do this (there are examples of this at a few places in the code).

Reply:
Done